Resources/Views/eventCard.leaf

```diff
@@ -1,5 +1,12 @@
 <a href="/event/#(event.shortname)">
-    <div class="conference-cell-no-image">
+    <div class="event-cell">
         <p>#(event.fullname)</p>
+        #if(event.startDate != nil || event.endDate != nil) {
```

Comment:
if you write if inside `<span class="date">`, we can omit the else clause.

Reply:
good point. fixed.